src/unix/bsd/mod.rs

```diff
@@ -324,18 +324,30 @@ f! {
         }
     }
 
-    pub fn WIFEXITED(status: ::c_int) -> bool {
-        (status & 0x7f) == 0
+    pub fn WIFSTOPPED(status: ::c_int) -> bool {
+        (status & 0o177) == 0o177
```

Comment:
For the bsd headers I see `#define WIFSTOPPED(x)	(_WSTATUS(x) == _WSTOPPED)`
Where `#define	_WSTOPPED	0177` and `#define	_WSTATUS(x)	(_W_INT(x) & 0177)`.